### ISSUE TITLE
No module named 'Mnist'

### ISSUE THREAD

**Issue by sara-kassani** [0x5eba/Skin-Cancer-Segmentation#1]
from Mnist.network import create_model
ModuleNotFoundError: No module named 'Mnist'

Hello, I'm trying to reproduce your code however I'm gwetting this error when I run python3 create_dataset.py.

**Comment by sara-kassani** [comment]
Thanks for your prompt answer. yeah I've changed the paths however there is a json file on line 37 of create_dataset.py that it seems not exit in this repo.

**Comment by 0x5eba** [comment]
It's because you have to download the dataset first of all. 
I added a script to download it easily.

I suggest you to download the entire repository again.

**Comment by sara-kassani** [comment]
Thank you again Sebastien. I've downloaded the repo again and run download_dataset.py. after downloading the data set, I've run create_dataset.py and i had this four options:
Press [m] to switch between displaying the selection, mask, or applied mask
Press [p] to print color statistics of current selection
Press [s] to save the currently displayed image
Press [q] or [esc] to close the window

I've choose the q and a npy file created in NumpyData directory. However I've noticed we don't have any grand truth or mask images in our dataset and when I run the main.py I've got the below error.

FileNotFoundError: [Errno 2] No such file or directory: 'NumpyData/ISIC_0009336_MASK.npy'

Thanks again for your time and consideration.
